configuration/configuration.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 OWASP Core Rule Set Project
+// SPDX-License-Identifier: Apache-2.0
+
+package configuration
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Configuration struct {
+	Patterns Patterns
+}
+
+type Patterns struct {
+	AntiEvasion              Pattern `yaml:"anti_evasion"`
+	AntiEvasionSuffix        Pattern `yaml:"anti_evasion_suffix"`
+	AntiEvasionNoSpaceSuffix Pattern `yaml:"anti_evasion_no_space_suffix"`
+}
+
+type Pattern struct {
+	Unix    string
+	Windows string
+}
+
+func New(directory string) *Configuration {
+	configFilePath := filepath.Join(directory, "toolchain.yaml")
```

Comment:
Why hardcoding here instead of passing a second filename arg?

Reply:
You're right.